lib/install.go

```diff
@@ -60,6 +60,11 @@ func getInstallLocation() string {
 
 	/* set installation location */
 	installLocation = usr.HomeDir + installPath
+	if usr.Uid == "0" {
+		// if we're root, use /usr/local/terraform
+		installLocation = "/usr/local/terraform/"
```

Comment:
As I commented in the issue, I'm not favorable to running `tfswitch` as root. But if this PR makes it through, imo `/opt/terraform` would be a better location according to [FHS](https://refspecs.linuxfoundation.org/fhs.shtml).

Reply:
happy to make it `opt` - though having worked on debian packages, and build linux distro's , FHS is a depressing document :)